onnx/backend/test/case/model/sequence.py

```diff
@@ -15,46 +14,46 @@
 from onnx.backend.test.case.model import expect
 
 
-def SequenceEmptyImpl() -> List[Optional[np.ndarray]]:
+def SequenceEmptyImpl() -> list[np.ndarray | None]:
```

Comment:
This change seems orthogonal to what the PR description says. Also: is this important? I guess `List` is compatible with more python versions than `list`? What do we gain by such changes?

Reply:
It's the fix from pyupgrade. Updated description. Since we have from `__future__ import annotations` it will work on python >=3.7

Reply:
Also `List` is deprecated from 3.9 so it would be good for us to take gradual steps toward conventions. ruff ensures we support the lowest python version specified (3.7 for now). When we drop support we will be able to change the config in pyproject.toml and have ruff automatically modernize the code